unit-tests/src/test/java/org/eclipse/collections/impl/list/immutable/AbstractImmutableListTestCase.java

```diff
@@ -429,28 +430,40 @@ public void toSortedListBy()
         Assert.assertEquals(this.classUnderTest(), sortedList);
     }
 
-    @Test
+    @Test(expected = UnsupportedOperationException.class)
     public void removeAtIndex()
     {
-        Verify.assertThrows(UnsupportedOperationException.class, () -> this.classUnderTest().castToList().remove(1));
+        this.classUnderTest().castToList().remove(1);
     }
 
-    @Test
+    @Test(expected = UnsupportedOperationException.class)
     public void set()
     {
-        Verify.assertThrows(UnsupportedOperationException.class, () -> this.classUnderTest().castToList().set(0, 1));
+        this.classUnderTest().castToList().set(0, 1);
     }
 
-    @Test
+    @Test(expected = UnsupportedOperationException.class)
     public void addAtIndex()
     {
-        Verify.assertThrows(UnsupportedOperationException.class, () -> this.classUnderTest().castToList().add(0, 1));
+        this.classUnderTest().castToList().add(0, 1);
     }
 
-    @Test
+    @Test(expected = UnsupportedOperationException.class)
     public void addAllAtIndex()
     {
-        Verify.assertThrows(UnsupportedOperationException.class, () -> this.classUnderTest().castToList().addAll(0, Lists.fixedSize.of()));
+        this.classUnderTest().castToList().addAll(0, Lists.fixedSize.of());
+    }
+
+    @Test(expected = UnsupportedOperationException.class)
+    public void sort()
+    {
+        this.classUnderTest().castToList().sort(Comparator.naturalOrder());
+    }
+
```

Comment:
When we have methods named sort() and sortThis(), as we probably do in this JUnit test, we should probably keep them together.

Reply:
Still relevant I think.